README.md

```diff
@@ -59,6 +59,8 @@ Model Optimizer is available for free for all developers on [NVIDIA PyPI](https:
 pip install "nvidia-modelopt[all]~=0.17.0" --extra-index-url https://pypi.nvidia.com
 ```
 
+Please note that unless you're an advanced user of this project, this step will not enable you to do almost anything, as multiple other requirements (and not only python packages) are required for things to work. You are advised to start with the [Docker](#docker) approach and then later adapt it to your specific environment.
```

Comment:
Can you share what issues you faced when getting it to work with pip?

I guess the complexity only comes in when you want to deploy to tensorrt-llm. If user wants to only use quantization, pruning, speculative decoding etc. features in pytorch then this pip install is very simple and works very smoothly.

Reply:
no, it's not smooth and it's working at all for just using modelopt. Please see the really long thread on slack that you were part of.

Please look at the dockerfile and you will see that it's not matching the pip install line.

Reply:
Can you invite me to the conversation. I don't have access

Reply:
My apologies, for some reason I thought you were already part of the conversation, but I was wrong. I have asked to add you to the channel and then will happily show you the conversation.